crates/rspack_plugin_runtime/src/runtime_module/utils.rs

```diff
@@ -346,3 +351,21 @@ fn test_get_undo_path() {
     "../../"
   );
 }
+
+#[test]
+fn test_stringify_chunks() {
+  #[rustfmt::skip]
```

Comment:
Actually add test at `packages/rspack/tests` is good enough, it's ok to add unit test at rust side, but since we are iterating fast about code, so unit test usually not keep long. And can you don't skip rustfmt is you wan't to keep this unit test?

Reply:
> Actually add test at `packages/rspack/tests` is good enough, it's ok to add unit test at rust side, but since we are iterating fast about code, so unit test usually not keep long. And could you make this not skip rustfmt is you still want to keep this unit test?

I’ll drop the unit test. It was useful before when I hadn’t switched to `serde_json`, but it’s not as relevant now that I’ve made the switch.

btw i add `#[rustfmt::skip]` there as i want the case array `pass` clean.